Storage/DiskImageStore.swift

```diff
@@ -44,50 +43,85 @@ open class DiskImageStore {
     }
 
     /// Gets an image for the given key if it is in the store.
-    open func get(_ key: String) -> Deferred<Maybe<UIImage>> {
-        return deferDispatchAsync(queue) { () -> Deferred<Maybe<UIImage>> in
-            if !self.keys.contains(key) {
-                return deferMaybe(DiskImageStoreErrorType(description: "Image key not found"))
-            }
+    open func getImageForKey(_ key: String) async throws -> UIImage {
```

Comment:
I think you can update the access modifiers to use public where needed instead of open as nobody currently sub classes this and if they need to in future they can change as needed.